src/inmanta/app.py

```diff
@@ -80,40 +80,23 @@ class MultiLineFormatter(colorlog.ColoredFormatter):
     def __init__(
         self,
         fmt: typing.Optional[str] = None,
-        datefmt: typing.Optional[str] = None,
-        style: _FormatStyle = "%",
+        *,
```

Comment:
the asterix means that the following argument should be set with the keyword right?

Reply:
Indeed. I added this to prevent incorrect usage based on the assumption that this class would have the same interface as its parent.